cmd/torpedo/torpedo.go

```diff
@@ -336,14 +339,29 @@ func (t *torpedo) validateContext(ctx *scheduler.Context) error {
 	}
 
 	if err := t.validateVolumes(ctx); err != nil {
+		generateSupportBundle(t, ctx)
 		return err
 	}
 	if err = t.s.WaitForRunning(ctx); err != nil {
+		generateSupportBundle(t, ctx)
```

Comment:
Add a warning log if failed to generate support bundle.

Reply:
Done.